codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -402,6 +417,12 @@ class FluentClientGenerator(
                             })?;
                         #{Ok}(#{TypedBox}::<#{OperationOutput}>::assume_from(output).expect("correct output type").unwrap())
                     }
+
+                    ##[doc(hidden)]
+                    // TODO(enableNewSmithyRuntime): Remove `async` once we switch to orchestrator
+                    pub async fn customize_orchestrator(self) -> #{CustomizableOperation} {
+                        #{CustomizableOperation} { fluent_builder: self, config_override: None, interceptors: vec![] }
```

Comment:
Will the initial config override be lost if I do something like the following?
```rust
client.some_operation()
    .config_override(some_config)
    .customize()
    .await
    .add_interceptor(some_interceptor)
    .send()
```

Reply:
Good question. The given snippet no longer complies after this PR, since there is no `config_override` on a fluent builder anymore. `config_override` is only accessible after you call `.customize`. Besides regular parameters for an operation input, I wanted to shove everything that's related to configuring a single operation invocation under `CustomizableOperation`. That being said, I'm curious to know what you think.

Reply:
Sounds good.